modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter-staff.po

```diff
@@ -0,0 +1,4 @@
+#: ../src/lightdm-gtk-greeter.c:1479
+msgid "Incorrect password, please try again"
+msgstr "Incorrect username or password,\n"
+"or you're not a staffer."
```

Comment:
Can we be more explicit about this? Something like...

"Sorry, but only OCF volunteer staffers can log in to this computer. Please use a different computer instead (or if you actually are staff, check your password)."

Anyone trying to log in to the computer is already clearly not very attentive, so I think making the message slightly longer and more direct would be a plus.